include/zephyr/bluetooth/iso.h

```diff
@@ -75,21 +75,21 @@ extern "C" {
 #define BT_ISO_DATA_PATH_VS_ID_MIN  0x01
 /** The maximum value for vendor specific data path ID */
 #define BT_ISO_DATA_PATH_VS_ID_MAX  0xFE
-/** Minimum controller delay in microseconds (0 s) */
+/** Minimum controller delay in microseconds (0 us) */
 #define BT_ISO_CONTROLLER_DELAY_MIN 0x000000
-/** Maximum controller delay in microseconds (4 s) */
+/** Maximum controller delay in microseconds (4.000.000 us) */
```

Comment:
Isn't this a number localization choice? Is there any guidelines documented @kartben ?

We have used `0.625 us`, `1.25 us` `1250 us`, `10000 microseconds` etc. in public APIs.

BLUETOOTH CORE SPECIFICATION Version 6.1 | Vol 3, Part C, Appendix A Timers and Constants, mostly uses seconds and millisecond units in US English formats? (I may be wrong here on which localization that it is).

```suggestion
/** Maximum controller delay in microseconds (4000000 us) */
```

Reply:
> Isn't this a number localization choice? Is there any guidelines documented @kartben ?

Asked about this on Discord as well (https://discord.com/channels/720317445772017664/884782246010175528/1393207293440426115), and there a few cases where the lack of defined formatting for large numbers can be an issue

> BLUETOOTH CORE SPECIFICATION Version 6.1 | Vol 3, Part C, Appendix A Timers and Constants, mostly uses seconds and millisecond units in US English formats

They usually mix units in the HCI spec, e.g. they may give a value (in microseconds) a range of 400 ms to 4 s 

Not sure if that is better or not, but I do like that when a value is in a specific unit (e.g. microseconds), then the range is also expressed in values of that unit, instead of mixing both microseconds and seconds. Some times the BT Core spec also uses 10 ms (centisecond) as the unit, but due to how rarely that unit is used, they always write "10 ms" instead, and arguably we should too, but I don't have a strong opinion here

Reply:
My question was related to the use of `.` versus `,` or none in the number system, example `4.000.000`.

Reply:
Created https://github.com/zephyrproject-rtos/zephyr/issues/94935

Reply:
Until #94935 gets some love I would recommend going with 400,000,000  i.e. American way of writing large numebrs given we have a high-level guideline that calls for using "American english".